README.md

```diff
@@ -8,5 +8,56 @@ This project creates the official [Apache Fluo][Fluo] docker image.
 
 * [Documentation] for running a Fluo application in Docker
 
+## Obtain the Docker image
+
+To obtain the docker image created by this project, you can either pull it from DockerHub at
+`apache/fluo` or build it yourself. To pull the image from DockerHub, run the command below:
+
+    docker pull apache/fluo
+
+While it is easier to pull from DockerHub, the image will default to the software versions below:
+
+| Software    | Version |
+|-------------|---------|
+| [Fluo]      | 1.2.0   |
+| [Accumulo]  | 1.8.1   |
+| [Hadoop]    | 2.7.3   |
+| [Zookeeper] | 3.4.9   |
+
+If these versions do not match what is running on your cluster, you should consider building
+your own image with matching versions. However, Fluo must be 1.2+.
+
+## Build the Docker image
```

Comment:
could add a comment that this section name is linked from web docs and needs to be stable....